modules/ROOT/pages/queries-aggregations/filtering.adoc

```diff
@@ -448,18 +447,46 @@ query {
 | Used in the case of `DateTime`, `LocalDateTime`, `LocalTime`, `Time`, and `Duration`.
 | `_MIN_EQUAL`, `_MIN_GT`, `_MIN_GTE`, `_MIN_LT`, `_MIN_LTE`, `_MAX_EQUAL`, `_MAX_GT`, `_MAX_GTE`, `_MAX_LT`, `_MAX_LTE`
 a|
+.Type definitions
+[source, graphql, indent=0]
+----
+type Event {
+    title: String!
+    startTime: DateTime!
+}
+----
+
+.Query
 [source, graphql, indent=0]
 ----
-{}
```

Comment:
Whaaaaaaaat, we just had empty queries here?!?!?!

Reply:
Nooo, it's because I decided to make that a table with an example for each item and those two didn't have an example when I merged the PR before, so Michael provided them to me later on and I used this PR to update